great_expectations/rule_based_profiler/attributed_resolved_metrics.py

```diff
@@ -35,10 +35,10 @@ def _detect_illegal_array_type_or_shape(values: MetricValues) -> bool:
                     return True
 
                 # Components of different lengths cannot be packaged into "numpy.ndarray" type (due to undefined shape).
-                if all(isinstance(value, list) for value in values):
+                if all(isinstance(value, (list, tuple)) for value in values):
                     values_iterator: Iterator = iter(values)
                     first_value_length: int = len(next(values_iterator))
-                    current_value: List[Any]
+                    current_value: Sized[Any]
```

Comment:
What is `Sized`?

Reply:
@NathanFarmer It is a generic `Type` that admits the `len(.)` operation.  I found it to be helpful.  Thanks!